addons/addon-base-raas/packages/base-raas-services/lib/schema/update-study.json

```diff
@@ -34,6 +34,9 @@
       "maxLength": 2048,
       "pattern": "^(arn:aws[a-zA-Z-]*:iam::[0-9]{12}:role[/]{1}[a-zA-Z0-9-]+)$"
     },
+    "tags": {
+      "type": "array"
+    },
```

Comment:
Is there a test for this?

Reply:
Updated tests in `study-service.test.js` to include `tags`. Without the change in this json the tests would fail.